texttospeech/list_voices/list_voices.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command list_voices lists the available text to speech voices.
```

Comment:
Package-level comments must be the line(s) immediately before the "package" declaration.